cmd/syncthing/gui.go

```diff
@@ -679,31 +699,6 @@ func (s *apiSvc) showGuiError(l logger.LogLevel, err string) {
 	guiErrorsMut.Unlock()
 }
 
-func (s *apiSvc) postSystemDiscovery(w http.ResponseWriter, r *http.Request) {
-	var qs = r.URL.Query()
-	var device = qs.Get("device")
-	var addr = qs.Get("addr")
-	if len(device) != 0 && len(addr) != 0 && s.discoverer != nil {
-		s.discoverer.Hint(device, []string{addr})
-	}
-}
-
-func (s *apiSvc) getSystemDiscovery(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ah..... Yeah, that may be worth preserving. Let me have a look at keeping that method around.